### PR TITLE
Completed webpack-intro exercise. -Lindsey Laveirge

### PR DESCRIPTION
**Pull request by llaveirge**
webpack-intro-vid

### CODE REVIEW

File: webpack-intro/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "lindsey",
+  "license": "ISC",
+  "dependencies": {
+    "jquery": "^3.6.0"
+  },
+  "devDependencies": {
+    "webpack-cli": "^4.9.2"
```
Comment (scbowler) [inline webpack-intro/package.json:17]: `webpack` should be listed under `devDependencies` here with `webpack-cli`

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just 1 update needed. Let me know if you have any questions.

File: webpack-intro/package.json (hunk 1 above)
Comment (llaveirge) [reply webpack-intro/package.json:17]: Thank you! I wasn't too clear on whether or not the install was just the cli or both, must have just been the cli!

**Comment by llaveirge** [comment]
image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**